equinix/services/fabricv4/api/cloud_routers_api.py

```diff
@@ -295,11 +295,12 @@ def _create_cloud_router_serialize(
 
 
         # set the HTTP header `Accept`
-        _header_params['Accept'] = self.api_client.select_header_accept(
-            [
-                'application/json'
-            ]
-        )
+        if 'Accept' not in _header_params:
+            _header_params['Accept'] = self.api_client.select_header_accept(
+                [
+                    'application/json'
+                ]
+            )
```

Comment:
These changes in the API classes are a result of https://github.com/OpenAPITools/openapi-generator/pull/19025, which enables overriding the `Accept` header